index.css

```diff
@@ -0,0 +1,197 @@
+@import url(https://fonts.googleapis.com/css?family=Oswald);
+
+.container {
+  font-family: Oswald;
+}
+
+a:hover {
+  color: #9068be;
+}
+
+nav {
+  background-color: #e62739;
+}
+
+
+ul.menu-options > li {
+  display: inline-block;
+  list-style-type: none;
+  text-align: center;
+  font-style: bold;
+  
+}
+
+#startrly {
+  margin-right: 400px;
+  margin-left: 1px;
+}
+
+#try {
+  margin-left: 300px;
+  margin-right: 300px;
+}
+
+.intro {
+  display: block;
+  text-align: center;
+  background-color: #6ed3cf;
+}
+
+.intro-heading {
+  font-size: 5em;
+}
+
+.intro-tagline {
+  font-size: 1.5em;
+  display: block;
+  margin-bottom: 30px;
+}
+
+.intro-action {
+  border: solid #9ea1a5 thin;
+  border-radius: 15px;
+  text-decoration: none;
+  color: black;
+  line-height: 5em;
+  padding: 13px;
+  font-size: 1.5em;
+}
+
+#line {
+  display: block;
+  height: 1px;
+  border: 0;
+  border-top: 1px solid #ccc;
+  margin: 1em 0;
+  padding-top: 0;
+}
+
+.quote {
+  font-style: italic;
+  font-size: 1.5em;
+  text-align: center;
+  padding: 2em;
+  background-color: #9068be;
+}
+
+.author {
+  display: inline;
+  float: right;
+  font-style: italic;
+  font-size: 20px;
+}
+
+#why-section {
+  background-color: #9ea1a5;
+}
+
+.why {
```

Comment:
Maybe there's an issue with class nesting here? Headers and paragraphs have some trouble with spacing. Maybe the solution is similar to what you discovered with my code! 👍🏼